vendor/assets/stylesheets/mailer.scss

```diff
@@ -0,0 +1,62 @@
+.mailer-content {
+  max-width: 900px;
+}
+
+.mailer-first-content {
+  background-color: #f7ede4;
+  max-width: 900px;
+}
+
+.message-title {
+  text-align: center;
```

Comment:
add .centered-paragraf  class to view, DRY

Reply:
Thanks, i fixed.